src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Stream.cs

```diff
@@ -394,18 +397,18 @@ public async ValueTask<int> ReadDataAsync(Memory<byte> buffer, CancellationToken
                 return bytesRead;
             }
 
-            private async ValueTask SendDataAsync(ReadOnlyMemory<byte> buffer)
+            private async ValueTask SendDataAsync(ReadOnlyMemory<byte> buffer, CancellationToken cancellationToken)
```

Comment:
ValueTask => Task

Reply:
The only usage of this function is in the implementation of `WriteAsync`, which is defined in the base class `BaseAsyncStream`, and returns ValueTask. I can rework that code to use Task instead of ValueTask if you'd like, but that seems like it's out of the scope of this PR.

Reply:
> but that seems like it's out of the scope of this PR

Ok.  We can fix it up separately.